src/config.rs

```diff
@@ -264,6 +270,11 @@ impl Config {
     pub fn loggers(&self) -> &[Logger] {
         &self.loggers
     }
+
+    /// Sets the logging level of the `Root`
```

Comment:
I'd probably instead add `root_mut` rather than directly setting the root level from here.

Reply:
Comment addressed.